kahuna/public/js/components/gr-collections-panel/gr-collections-panel.js

```diff
@@ -0,0 +1,112 @@
+import angular from 'angular';
+
+import '../../services/panel';
+import '../../services/api/collections-api';
+import '../../directives/gr-auto-focus';
+
+import './gr-collections-panel.css!';
+import nodeTemplate from './gr-collections-panel-node.html!text';
+
+export var grCollectionsPanel = angular.module('grCollectionsPanel', [
+    'kahuna.services.panel']);
+
+grCollectionsPanel.controller('GrCollectionsPanelCtrl', [
+    'panelService',
+    '$rootScope',
+    'collections',
+    function (panelService, $rootScope, collections) {
+
+    const ctrl = this;
+    const panelName = 'gr-collections-panel';
+
+    ctrl.isVisible = false;
+
+    panelService.addPanel(panelName, false);
+    panelService.available(panelName, true);
+
+    $rootScope.$on(
+        `ui:panels:${panelName}:updated`,
+        () => {
+            ctrl.isVisible = panelService.isVisible(panelName);
+        }
+    );
+
+    collections.getCollections().then(collections => {
+        ctrl.collections = collections.data.children;
+    });
```

Comment:
Not urgent, but we should really add error handler in such places to display an error message ("oops, collections not available, please try again later") in case the collections service is not available.